content/03.text-mining.md

```diff
@@ -12,9 +12,37 @@
 
 ### Text Mining for Relationship Extraction
 
-#### Rule-Based Natural Language Processing
+#### Rule-Based Relationship Extraction
 
-1. Mention papers on hand written rules and expressions
+Rule-based extraction consists of identifying sentences that contain important keywords or grammatical patterns that allude to relationships of interest. 
+These keywords are established via expert knowledge or pre-existing ontologies.
+Grammatical patterns are mainly constructed via experts curating part of speech tags or dependency path trees.
+Most text mining approaches use the above methods to generate features for machine learning algorithms and are discussed in later sections.
+For this section we focus on approaches that mainly use rule based extraction to detect sentences that assert a relationship.
+
+Rules that use grammatical patterns can be used to simplify sentences for easy extraction.
```

Comment:
`Rules that use` -> Do you mean `Rules for` or something else? I don't entirely understand this sentence as it is.